### PR TITLE
feat(rms): add new resource organizational policy assignment

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

add new resource organizational policy assignment

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
go test ./huaweicloud/services/acceptance/rms -v -run TestAccOrganizationalPolicyAssignment_basic -timeout 360m
=== RUN   TestAccOrganizationalPolicyAssignment_basic
=== PAUSE TestAccOrganizationalPolicyAssignment_basic
=== CONT  TestAccOrganizationalPolicyAssignment_basic
--- PASS: TestAccOrganizationalPolicyAssignment_basic (90.10s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rms       92.115s

go test ./huaweicloud/services/acceptance/rms -v -run TestAccOrganizationalPolicyAssignment_custom -timeout 360m
=== RUN   TestAccOrganizationalPolicyAssignment_custom
=== PAUSE TestAccOrganizationalPolicyAssignment_custom
=== CONT  TestAccOrganizationalPolicyAssignment_custom
--- PASS: TestAccOrganizationalPolicyAssignment_custom (93.16s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rms       95.051s
```

### CODE REVIEW

File: docs/resources/rms_policy_organizational_assignment.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  + **One_Hour**
+  + **Three_Hours**
+  + **Six_Hours**
+  + **Twelve_Hours**
+  + **TwentyFour_Hours**
+
+  Most one of `period` and `policy_filter` can be configured.
+
+* `policy_filter` - (Optional, List) Specifies the configuration used to filter resources.  
+  The [object](#rms_policy_filter) structure is documented below.
```
Comment (houpeng80) [inline docs/resources/rms_policy_organizational_assignment.md]:
```suggestion
  The [policy_filter](#rms_policy_filter) structure is documented below.
```

File: huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+			"resource_type":     filterMap["resource_type"],
+			"resource_id":       filterMap["resource_id"],
+			"tag_key":           filterMap["tag_key"],
+			"tag_value":         filterMap["tag_value"],
+		},
+	}
+}
+
+func flattenOrganizationalPolicyParameters(parameters interface{}) (map[string]interface{},
+	error) {
```
Comment (houpeng80) [inline huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go]:
```suggestion
func flattenOrganizationalPolicyParameters(parameters interface{}) (map[string]interface{}, error) {
```

File: huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+	if parameters == nil {
+		return nil, nil
+	}
+
+	result := make(map[string]interface{})
+	for k, v := range parameters.(map[string]interface{}) {
+		val := v.(map[string]interface{})
+		jsonBytes, err := json.Marshal(val["value"])
+		if err != nil {
+			return nil, fmt.Errorf("generate json string failed: %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go]: it's better to add the key in error message
```suggestion
			return nil, fmt.Errorf("generate json string of %s failed: %s", key, err)
```

File: docs/resources/rms_policy_organizational_assignment.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/rms_policy_organizational_assignment.md]: fixed

File: huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go]: fixed

File: docs/resources/rms_policy_organizational_assignment.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  organization_id = data.huaweicloud_organizations_organization.test.id
+  name            = var.policy_assignment_name
+  description     = "This is a custom policy assignment."
+  function_urn    = "${huaweicloud_fgs_function.test.urn}:${huaweicloud_fgs_function.test.version}"
+  period          = "TwentyFour_Hours"
+
+  parameters = {
+    string_test = "\"string_value\""
+    array_test  = "[\"array_element\"]"
+    object_test = "{\"terraform_version\":\"1.xx.x\"}"
```
Comment (ShiChangkuo) [inline docs/resources/rms_policy_organizational_assignment.md]: how about using `jsonencode`?
```suggestion
    object_test = jsonencode({"terraform_version": "1.xx.x"})
```

File: docs/resources/rms_policy_organizational_assignment.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+The following arguments are supported:
+
+* `organization_id` - (Required, String, ForceNew) Specifies the ID of the organization.  
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the organizational policy assignment.  
+  The valid length is limited from `1` to `60`, only letters, digits, hyphens (-) and underscores (_) are allowed.  
+  Change this parameter will create a new resource.
+
+* `excluded_accounts` - (Optional, String) Specifies the excluded accounts of the organizational policy assignment.
```
Comment (ShiChangkuo) [inline docs/resources/rms_policy_organizational_assignment.md]: this parameter should be a list of string?

File: huaweicloud/services/acceptance/rms/resource_huaweicloud_rms_organizational_policy_assignment_test.go [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getOrganizationalPolicyAssignmentResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/rms/resource_huaweicloud_rms_organizational_policy_assignment_test.go:68]: we should add a pre-check for Organizations service, please add acceptance.TestAccPreCheckOrganizationsOpen

File: huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go (hunk 3 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/rms/resource_huaweicloud_rms_organizational_policy_assignment.go]: fixed

File: docs/resources/rms_policy_organizational_assignment.md (hunk 4 above)
Comment (Jason-Zhang9309) [reply docs/resources/rms_policy_organizational_assignment.md]: fixed

File: docs/resources/rms_policy_organizational_assignment.md (hunk 5 above)
Comment (Jason-Zhang9309) [reply docs/resources/rms_policy_organizational_assignment.md]: fixed

File: huaweicloud/services/acceptance/rms/resource_huaweicloud_rms_organizational_policy_assignment_test.go (hunk 6 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/rms/resource_huaweicloud_rms_organizational_policy_assignment_test.go:68]: fixed

**Pull request merged**